docs/04-operation-guides/troubleshooting/00-eventing/evnt-01-eventing-troubleshooting.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Kyma Eventing - Basic Diagnostics
+---
+
+## Symptom(s)
+
+- You publish an event but the event is not received by the subscriber.
+
+- Subscription is not in ready state.
+
+- You are unable to publish events.
+
+## Remedy
+
+Follow these steps to detect the source of the problem:
+
+### Step 1: Check the status of Check the Eventing backend CR
+
+Check the Eventing backend CR, if the field `EVENTINGREADY` is true:
+   
+```bash
+kubectl -n kyma-system get eventingBackend
+```
+If `EVENTINGREADY` is false, check the exact reason of the error in the status of the CR by running the command:
+
+```bash
+kubectl -n kyma-system get eventingBackend eventing-backend -o yaml
+```
+
+### Step 2: Check the status of the Subscription
+
+Check the Subscription, if it is ready. Run the command:
+
+```bash
+kubectl -n {NAMESPACE} get subscription {NAME}
```

Comment:
Please let's change `subscription ` to `subscriptions.eventing.kyma-project.io`. Also applies in multiple places in the document.

Reply:
done